isis/src/qisis/objs/JigsawSetupDialog/JigsawSetupDialog.h

```diff
@@ -156,10 +157,11 @@ namespace Isis {
 
 
     private:
-    void makeReadOnly()    ;
+    void makeReadOnly();
     void fillFromSettings(const BundleSettingsQsp settings);
     void showTargetParametersGroupBox();
     void hideTargetParametersGroupBox();
+    void updateBundleObservationSolveSettings(BundleObservationSolveSettings &boss);
 
     void createObservationSolveSettingsTreeView();
 
```

Comment:
I would add a history comment about adding updateBundleObservationSettings and what it does